<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cancel_Seat_2</class>
 <widget class="QDialog" name="Cancel_Seat_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cancel Selection</string>
  </property>
  <widget class="QPushButton" name="btnCancel2">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Seat  :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCancel">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>66</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblSeat">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>184</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee UI Semilight</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
